src/resources/views/crud/inc/export_buttons.blade.php

```diff
@@ -1,12 +1,13 @@
 @if ($crud->exportButtons())
-  @basset('https://cdn.datatables.net/buttons/1.5.6/js/dataTables.buttons.min.js')
-  @basset('https://cdn.datatables.net/buttons/1.5.6/js/buttons.bootstrap4.min.js')
+  @basset('https://cdn.datatables.net/buttons/2.3.3/css/buttons.bootstrap5.min.css')
+  @basset('https://cdn.datatables.net/buttons/2.3.3/js/dataTables.buttons.min.js')
+  @basset('https://cdn.datatables.net/buttons/2.3.3/js/buttons.bootstrap5.min.js')
   @basset('https://cdnjs.cloudflare.com/ajax/libs/jszip/2.5.0/jszip.min.js')
   @basset('https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.18/pdfmake.min.js')
   @basset('https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.1.18/vfs_fonts.js')
-  @basset('https://cdn.datatables.net/buttons/1.5.6/js/buttons.html5.min.js')
-  @basset('https://cdn.datatables.net/buttons/1.5.6/js/buttons.print.min.js')
-  @basset('https://cdn.datatables.net/buttons/1.5.6/js/buttons.colVis.min.js')
```

Comment:
This seems like a general change, not specific to this theme branch. So let's do it on the `v6` branch.

Reply:
It's not - it's a Bootstrap 5 change, so it's ok here.